<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Analysis of magnetic variations</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="signalGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Signal</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="signalGridLayout" rowstretch="0,0,0,0">
          <property name="spacing">
           <number>1</number>
          </property>
          <item row="1" column="3">
           <widget class="QLabel" name="sizeLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>2^1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="2">
           <widget class="QLabel" name="offsetLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Time offset:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QScrollBar" name="offsetHorizontalScrollBar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="scalogramGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Scalogram</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="scalogramGridLayout" rowstretch="0,0,0,0">
          <property name="spacing">
           <number>1</number>
          </property>
          <item row="1" column="3">
           <widget class="QLabel" name="notesLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="2">
           <widget class="QLabel" name="scaleLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Largest scale</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>Notes:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QScrollBar" name="scaleHorizontalScrollBar">
            <property name="minimum">
             <number>4</number>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="toolGroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string>Parametrs</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Wavelet:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="waveletComboBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Order:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="orderSpinBox">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Omega0:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="omega0SpinBox">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Range  from</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="minHspinBox">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxHspinBox">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionDownload"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSave_image_signal_as"/>
    <addaction name="actionSave_scalogram_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionDataHeader"/>
    <addaction name="actionPlot_periodogram"/>
    <addaction name="actionPlot_scalegram"/>
    <addaction name="actionDetrend"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionDataHeader">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Data header</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionPlot_signal">
   <property name="text">
    <string>Plot signal</string>
   </property>
  </action>
  <action name="actionSave_image_signal_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save signal as...</string>
   </property>
  </action>
  <action name="actionSave_scalogram_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save scalogram as...</string>
   </property>
  </action>
  <action name="actionPlot_periodogram">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Plot periodogram</string>
   </property>
  </action>
  <action name="actionPlot_scalegram">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Plot scalegram</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="text">
    <string>Download...</string>
   </property>
  </action>
  <action name="actionDetrend">
   <property name="text">
    <string>Detrend</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
